### DISCUSSION TITLE
Need Help Updating Pocketbase Version in Digital Ocean

### DISCUSSION THREAD

**Discussion by suelitonx** [pocketbase/pocketbase#4273] [Q&A] [answered]
Hello everyone,

I recently deployed Pocketbase using the link "https://marketplace.digitalocean.com/apps/pocketbase" with version 0.18.5. Now, I am looking for guidance on how to update it to the latest version without creating a new Droplet.

I have checked the official documentation, but I couldn't find specific instructions on the update process. The deployment was done on Digital Ocean, and I want to make sure I follow the correct steps to update Pocketbase without starting over.

I discovered that the files are located in "/opt/pocketbase/pb_data," and now I need assistance on how to manually update the Pocketbase version. If anyone has experience with updating Pocketbase or similar deployments on Digital Ocean and can guide me through the process of manual update, your help would be greatly appreciated!

If possible, please provide step-by-step instructions or point me to any relevant documentation.

Thank you. Note: I'm Brazilian, so forgive my English

**Comment by ganigeorgiev** [comment, marked as answer]
I haven't tried the above DO 1-Click app but it should be enough to SSH into your droplet and run something like:
```sh
systemctl stop pocketbase

# by default it will create a backup of your pb_data (you can disable this with `--backup=0`)
/opt/pocketbase/pocketbase update

systemctl start pocketbase
```

Excluding the refactored logs, there shouldn't be any other major breaking changes between v0.18-v0.21 but nevertheless you could have a quick look at the [CHANGELOG.md](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md).

Alternatively, you can download locally the binary for your droplet platform and scp/rsync it to replace the `/opt/pocketbase/pocketbase` file and then restart the service (_this will allow you to also test the changes locally first_).
